docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

```diff
@@ -77,6 +77,10 @@ If you manage your own native projects, you will need to manually apply any chan
 
 Each SDK announcement blog post contains deprecations, breaking changes, and anything else that might be unique to that particular SDK version. When upgrading, be sure to check these out to make sure you don't miss anything.
 
+### SDK 50 (alpha)
```

Comment:
heads up - we're using the name "canary" to refer to our continuous pre-releases prior to beta

Reply:
That showed up here from fixing conflicts from main, which references SDK 50 alpha here. I just updated it to `(canary)`